docs/src/monitoring.md

```diff
@@ -359,7 +359,7 @@ go_threads 18
 ### User defined metrics
 
 This feature is currently in *beta* state and the format is inspired by the
-[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/master/queries.yaml) <!-- wokeignore:rule=master -->
+[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/v0.12.1/queries.yaml)
```

Comment:
why specifically 0.12.1? There seem to be more recent versions.

Reply:
OK, answered my own question. The more modern versions have no content 🤔

Reply:
Wow ... ultimately the Prometheus exporter has adopted our preferred way to write queries in Go directly. Look:

https://github.com/prometheus-community/postgres_exporter/pull/801#issuecomment-1588258934

Reply:
A new project has been created for this purpose, and database independent: https://github.com/burningalchemist/sql_exporter